cmd/zar/zq/command.go

```
@@ -90,12 +78,12 @@ func (c *Command) Run(args []string) error {
 	}
 	// XXX this is parallelizable except for writing to stdout when
 	// concatenating results
-	return archive.Walk(ark, func(zardir string) error {
+	return archive.Walk(ark, func(zardir iosrc.URI) error {
 		inputs := args
 		var query ast.Proc
 		var err error
 		first := archive.Localize(zardir, inputs[0])
-		if first != "" && fileExists(first) {
+		if ok, _ := iosrc.Exists(first); ok {
```

any reason to ignore the error here?

---

Actually, no. Will fix.